mono/mini/interp-stubs.c

```diff
@@ -0,0 +1,180 @@
+#include <config.h>
```

Comment:
Why was this moved to mini/ ?

Reply:
I want everything out of the `interp/` directory that should not end up in the library. In order to avoid ifdef hell, the interp-stubs are now always part of the runtime, so there's a fallback for the callbacks available.